README.md

```diff
@@ -152,12 +152,12 @@ the 2023 assessment model.
 | Percent Population Mobility, Moved From Within Same County in Past Year | acs5           | numeric   |                       |
 | Longitude                                                               | loc            | numeric   |                       |
 | Latitude                                                                | loc            | numeric   |                       |
+| Municipality Name                                                       | loc            | character |                       |
```

Comment:
This reverses a change that was made (somewhat mysteriously) in https://github.com/ccao-data/model-condo-avm/pull/4 when adding the `var_name_model != "loc_tax_municipality_name"` check ended up moving this row lower in the table.